fe/fe-core/src/main/java/com/starrocks/load/Load.java

```diff
@@ -807,6 +807,7 @@ private static Expr transformHadoopFunctionExpr(Table tbl, String columnName, Ex
                 FunctionCallExpr newFn = new FunctionCallExpr("if", exprs);
                 return newFn;
             } else if (funcName.equalsIgnoreCase(FunctionSet.STRFTIME)) {
+                // TODO(yan): we need to do some cast here?
```

Comment:
@Seaven do you think it's necessary to do cast right here? Or FE will handle type conversion ?

Reply:
I don't konw this logical, but it's looks error to construct function directly .....may be you need @chaoyli